scripts/versions.sh

```diff
@@ -29,8 +29,9 @@ function extract_commit() {
   echo "$version"
 }
 
-# AWM_RELAYER_VERSION is needed for the docker run setup, but is not a go module dependency.
-AWM_RELAYER_VERSION=${AWM_RELAYER_VERSION:-'v1.0.0'}
+# ICM_SERVICES_VERSION is needed for the E2E tests but is not a direct dependency since that would create a circular dependency.
+# ICM_SERVICES_VERSION=${ICM_SERVICES_VERSION:-'signature-aggregator-v1.0.0-rc.0'}
+ICM_SERVICES_VERSION=${ICM_SERVICES_VERSION:-'7d1e095dc84913d28c9c82806f240595e8cd4c8b'}
```

Comment:
Right now this is pointing to a commit on https://github.com/ava-labs/icm-services/pull/616 branch. Should replace when we have the next tag